analyses/models.py

```diff
@@ -694,6 +720,7 @@ class PipelineVersions(models.TextChoices):
     )
 
     class AnalysisStates(FutureStrEnum):
+        # TODO: this is pipeline specific - I think we need to move elsewhere or refactor (mbc)
```

Comment:
My gut says we should have a strictly keyed high level generic state (even fewer than this set), and then a freely-keyed one for the pipeline specifics.